### PR TITLE
Convert MSA output into vectors of gapped sequence

### PR DESCRIPTION
**Pull request by joergi-w**

### CODE REVIEW

File: include/seqan3/alignment/multiple/align_multiple.hpp [hunk 1]
```diff
@@ -54,7 +59,7 @@ auto seqan2_msa_configuration()
     msaOpt.sc.data_gap_open = -13;  // tcoffee app default
     msaOpt.sc.data_gap_extend = -1; // tcoffee app default
     msaOpt.sc.data_match = 5;       // tcoffee app default
-    msaOpt.sc.data_match = -4;      // tcoffee app default
+    msaOpt.sc.data_mismatch = -4;   // tcoffee app default
```
Comment (smehringer) [inline include/seqan3/alignment/multiple/align_multiple.hpp:62]: ops :D

File: include/seqan3/alignment/multiple/align_multiple.hpp [hunk 2]
```diff
@@ -106,14 +111,19 @@ auto align_multiple(std::vector<range_t> const & input)
...
     }
 
-    // fake result
-    seqan3::gap g{};
-    std::vector<std::vector<seqan3::gapped<seqan3::dna4>>> output
+    std::string mat;
+    seqan::convertAlignment(gAlign, mat);
+
+    using alph_type = seqan3::gapped<seqan3::dna4>;
+    std::vector<std::vector<alph_type>> output{seqan::length(seqan::stringSet(gAlign))};
```
Comment (smehringer) [inline include/seqan3/alignment/multiple/align_multiple.hpp]:
```suggestion
    using gapped_alphabet_type = seqan3::gapped<seqan3::dna4>;
    std::vector<std::vector<gapped_alphabet_type>> output{seqan::length(seqan::stringSet(gAlign))};
```

**Review by smehringer — COMMENTED** [review]
Nice :) 

Just a small suggestion.

File: include/seqan3/alignment/multiple/align_multiple.hpp [hunk 3]
```diff
@@ -106,14 +111,19 @@ auto align_multiple(std::vector<range_t> const & input)
...
+    size_t ali_len = mat.size() / output.size();
+
+    auto iter = output.begin();
+    for (auto && gapped_seq : mat | seqan3::views::char_to<alph_type> | seqan3::views::chunk(ali_len))
```
Comment (joergi-w) [inline include/seqan3/alignment/multiple/align_multiple.hpp]:
```suggestion
    for (auto && gapped_seq : mat | seqan3::views::char_to<gapped_alphabet_type> | seqan3::views::chunk(ali_len))
```

**Pull request merged**